src/transaction/types.ts

```diff
@@ -85,6 +98,23 @@ type PaymasterService = {
   url: string;
 };
 
+export type sendBatchedTransactionsParams = {
+  capabilities?: WalletCapabilities;
+  // biome-ignore lint: cannot find module 'wagmi/experimental/query'
```

Comment:
It's fine for now, but can we ask the Wagmi team to help export those types, please.